src/main/java/echoServer/ClientSocketWrapperInterface.java

```diff
@@ -0,0 +1,7 @@
+package echoserver;
+public interface ClientSocketWrapperInterface {
```

Comment:
It's not necessary to include `Interface` as part of the interface's name. Instead, the interface should be well-named (which I think `SocketWrapper` is). One alternative is to append "I" to the beginning of the name (e.g. `ISocketWrapper`), but the internet definitely doesn't agree on whether or not this is a good practice. I don't mind either way as long as your interfaces are named consistently.